### DISCUSSION TITLE
What should I do if there is a space in the uri when starting Android deeplink?

### DISCUSSION THREAD

**Discussion by llwwbb** [appium/appium#17505] [Q&A] [answered]
[apk-utils.js](https://github.com/appium/appium-adb/blob/c0ea020aa1facf9885633682cd7be0e1f9d2ad78/lib/tools/apk-utils.js#L79)

looks like it only escape on windows, but not linux

Is it a good idea to wrap the uri parameter in quotes directly?

**Comment by jlipps** [comment, marked as answer]
uris also can't have spaces the wild, right? did you try urlencoding?

**Reply by llwwbb** [reply]
yes, urlencoding works
